src/main/java/nl/tudelft/dnainator/ui/views/View.java

```diff
@@ -21,6 +21,9 @@
  */
 public class View extends Pane {
 	private static final double SCALE = .1;
+	private static final double ZOOM_FACTOR = 1e-3;
+	private static final int DEFAULT_ZOOM_IN = 55;
```

Comment:
Different zoom levels are used because for some reason this is inconsistent. On @pvanagtmaal's setup, these zoomlevels make it look somewhat consistent. Not so much on mine, but we're assuming this is OS/display resolution dependent.